lib/config/ChainConfigManager.ts

```diff
@@ -0,0 +1,293 @@
+import { ChainId } from '@uniswap/sdk-core';
+import { RoutingType } from '../constants';
+
+type ChainConfig = {
+  routingTypes: (
+    | {
+        routingType: RoutingType.CLASSIC | RoutingType.DUTCH_V2 | RoutingType.RELAY;
+        skipRFQ?: boolean;
+        priceImprovementBps?: number;
+        stdAuctionPeriodSecs?: number;
+        deadlineBufferSecs?: number;
+      }
+    | {
+        routingType: RoutingType.DUTCH_LIMIT;
+        skipRFQ?: boolean;
+        priceImprovementBps?: number;
```

Comment:
what does this priceImprovementBps refer to?

Reply:
For Open Orders (aka Synthetic orders) we take a classic order and add 1 bps of "price improvement". 
https://github.com/Uniswap/unified-routing-api/blob/b1f38fd1b922b57423b77156d4cbd7619aeda255/lib/entities/quote/DutchQuote.ts#L232

This config param allows us to change that per chain. On L2s we may want to set a larger range since we have more granularity (faster block rate).

Reply:
Ah gotcha, it'll be helpful to change that for L2s